distribution/tools/fips-demo-installer-cli/src/test/java/org/opensearch/tools/cli/fips/truststore/TrustStoreServiceTests.java

```diff
@@ -28,15 +28,22 @@
 import static org.opensearch.tools.cli.fips.truststore.ConfigureSystemTrustStore.findPKCS11ProviderService;
 
 public class TrustStoreServiceTests extends OpenSearchTestCase {
-    @ClassRule
-    public static TemporaryFolder tempFolder = new TemporaryFolder();
+
+    private static Path sharedTempDir;
```

Comment:
The changes here look like they are reverting some of the changes made in https://github.com/opensearch-project/OpenSearch/pull/19806 to use a dedicated tmp folder per suite. Can we use the model of dedicated tmp folder for a suite to better isolate the tests?